src/dblib/unittests/numeric.c

```diff
@@ -172,7 +172,7 @@ main(int argc, char **argv)
 {
 	read_login_info(argc, argv);
 
-	dbsetversion(DBVERSION_100);
+	/* dbsetversion(DBVERSION_100); */
```

Comment:
FreeTDS is tested with success against MSSQL, can you elaborate the problem?
I think this is here to make the test work using Sybase libraries.

Reply:
With cb97ca1 merged, this `dbsetversion` call causes FreeTDS to insist on using TDS 5.0, which only Sybase supports.  I suppose I should squash those changes into a single patch.